src/components/charts/charts.css

```diff
@@ -347,3 +348,20 @@
 .form__error.flex {
     display: flex;
 }
+
+.filter-tab {
+    overflow-y: auto;
+    overflow-x: hidden;
+    height: calc(100% - 64px);
+}
+
+.chart-list-view {
+    max-width: 776px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.create-button {
```

Comment:
we can minimize this css  using `bcn-0 en-2 bw-1`

Reply:
Done